cmd/state/main.go

```diff
@@ -165,7 +147,26 @@ func run(args []string, out output.Outputer) (int, error) {
 
 	// Run the actual command
 	cmds := cmdtree.New(primer.New(pj, out, authentication.Get(), prompter, sshell, conditional))
-	err = cmds.Execute(args[1:])
 
+	// Auto update to latest state tool version, only runs once per day
```

Comment:
Let's do the cmd check here. The AutoUpdate or Check code shouldn't need to have the command passed to them. (See my comment about unifying the checks also).